### ISSUE TITLE
Install fails in macOS Big Sur (Intel)

### ISSUE THREAD

**Issue by marcolabreu** [minoki/purescript-tsd-gen#9]
Unable to install **purescript-tsd-gen** in an Intel MacBook Pro running macOS Big Sur 11.4

The complete log from `stack install` is 

```bash
purescript        > configure
purescript        > Configuring purescript-0.13.8...
purescript        > build
purescript        > Preprocessing library for purescript-0.13.8..
purescript        > Building library for purescript-0.13.8..
purescript        > [  1 of 163] Compiling Control.Monad.Logger
purescript        > [  2 of 163] Compiling Control.Monad.Supply
purescript        > [  3 of 163] Compiling Control.Monad.Supply.Class
purescript        > [  4 of 163] Compiling Language.PureScript.Comments
purescript        > <command line>: can't load framework: Cocoa (not found)
Progress 1/2

--  While building package purescript-0.13.8 (scroll up to its section to see the error) using:
      /Users/<user_name>/.stack/setup-exe-cache/x86_64-osx/Cabal-simple_mPHDZzAJ_2.4.0.1_ghc-8.6.5 --builddir=.stack-work/dist/x86_64-osx/Cabal-2.4.0.1 build --ghc-options " -fdiagnostics-color=always"
    Process exited with code: ExitFailure 1
```

**Comment by minoki** [comment]
That's because the version of GHC is old. GHC's issue is https://gitlab.haskell.org/ghc/ghc/-/issues/18446 and this is fixed in GHC 8.10.3 or later (or stackage lts-17.x).

Maybe using a newer GHC means upgrading purescript? Sorry for being late for catching-up my tool to latest purescript version.

**Comment by minoki** [comment]
Closing.
